setup_proxmox.sh

```
@@ -0,0 +1,630 @@
+    elif command -v yum &> /dev/null; then
+        yum update -y  
+        yum install -y curl wget git unzip gcc gcc-c++ make
+    else
+        error "Desteklenmeyen paket yöneticisi. Ubuntu/Debian/CentOS/RHEL kullanın."
+    fi
+    
+    log "Sistem paketleri güncellendi."
+}
+
+# Install Node.js 20.x
+install_nodejs() {
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)